Dnn.AdminExperience/Dnn.PersonaBar.Extensions/Services/SiteSettingsController.cs

```diff
@@ -2095,8 +2095,8 @@ public HttpResponseMessage GetIgnoreWords(int? portalId, string cultureCode)
                 {
                     return Request.CreateErrorResponse(HttpStatusCode.Unauthorized, AuthFailureMessage);
                 }
-
-                var language = LocaleController.Instance.GetLocale(pid, cultureCode);
+                var language = string.IsNullOrEmpty(cultureCode)
+                    ? LocaleController.Instance.GetCurrentLocale(pid) : LocaleController.Instance.GetLocale(pid, cultureCode);
```

Comment:
You can move this logic into LocaleController with name "GetLocaleOrCurrent(pid, cultureCode)"

public Locale GetLocaleOrCurrent(int portalID, string cultureCode) =>
    string.IsNullOrEmpty(cultureCode) ? LocaleController.Instance.GetCurrentLocale(portalID) : LocaleController.Instance.GetLocale(portalID, cultureCode);